G1ANT.Addon.Core/Commands/ContainsCommand.md

```diff
@@ -0,0 +1,35 @@
+```
+
+## Description
+
+This command checks if a specified dictionary contains a given key.
+
+| Argument       | Type                                                         | Required | Default Value                                               | Description                                                  |
+| -------------- | ------------------------------------------------------------ | -------- | ----------------------------------------------------------- | ------------------------------------------------------------ |
+|`dictionary`| [dictionary](G1ANT.Language/G1ANT.Language/Structures/DictionaryStructure.md) | yes |  | Dictionary to be checked for a presence of a specified key |
+|`key`| [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | yes | | Key to look for in a dictionary |
+| `result`       | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       | `♥result`                                                   | Name of a variable where the command result (`true` or `false`) will be stored |
+| `if`           | [bool](G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
+
+For more information about `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
+
+## Example
+
+In this simple example a dictionary entry is created containing `username:g1ant` key-value pair, then the robot checks whether this dictionary contains a key named `username`. Since it does, a dialog box displays *true* as the `contains` command result:
+
+```G1ANT
+♥dictionary = ⟦dictionary⟧username❚g1ant
```

Comment:
it's correct but I think it will be better if you add some more values since dictionary is not a single key-value pair but a list of key-value pairs. So I suggest to:
```♥dictionary = ⟦dictionary⟧username❚g1ant❚password❚mypass123```
and check for password key presence 😄

Reply:
you're right, I'll rewrite that